src/components/input-date-picker/input-date-picker.tsx

```diff
@@ -522,6 +529,7 @@ export class InputDatePicker
                   intlYear={this.intlYear}
                   max={this.max}
                   maxAsDate={this.maxAsDate}
+                  messageOverrides={this.messageOverrides}
```

Comment:
Can you update the conventions doc and/or util doc about this scenario (non-localized/t9n component forwarding props)?